src/components/Alphabet/AlphabetPresentationSelected.js

```diff
@@ -111,12 +114,18 @@ function AlphabetPresentationSelected({
       )}
       {generalNote?.length > 0 && (
         <div className="flex-col justify-center mx-auto my-5 w-3/4">
-          <h2 className="sm:text-2xl font-medium text-xl text-center text-blumine-800 p-3">
-            Notes
-          </h2>
+          <h2 className={labelStyling}>Notes</h2>
           <p className="text-center">{generalNote}</p>
         </div>
       )}
+      {relatedDocuments?.length > 0 && (
+        <div className="flex-col justify-center mx-auto my-5 w-3/4">
+          <RelatedDocumentsList.Presentation
+            documents={relatedDocuments}
+            labelStyling={labelStyling}
+          />
```

Comment:
I'm noticing that in this context, it would make sense for the Related Documents header to be an h2. Should the header perhaps be pulled out of the reusable component so it can be customized?

Reply:
This approach would also remove the need to pass a `labelStyling` prop.

Reply:
I agree! pulled out the header and removed the labelstyling prop